update.php

```diff
@@ -431,3 +431,29 @@ function update_1347()
 
 	return Update::SUCCESS;
 }
+
+function pre_update_1348()
+{
+	DBA::insert('contact', ['nurl' => '']);
+	DBA::update('contact', ['id' => 0], ['id' => DBA::lastInsertId()]);
```

Comment:
You didn't check if the previous insert was successful, which means `DBA::lastInsertId()`'s behavior is uncertain, potentially sending a random contact into the void.

Why not insert the `id =-0` value explicitly like this: `DBA::insert('contact', ['id' => 0, 'nurl' => '']);`?

Reply:
inserting "0" triggers the auto increment.